packages/terra-clinical-item-collection/tests/nightwatch/item-collection/ItemCollectionTests.jsx

```diff
@@ -7,19 +7,14 @@ const ItemCollectionTests = () => (
   <div>
     <ul>
       <li><Link to="/tests/item-collection-tests/default">Item Collection - Default</Link></li>
-      <li><Link to="/tests/item-collection-tests/tiny-breakpoint">Item Collection - Tiny Breakpoint</Link></li>
```

Comment:
These breakpoint tests were testing that the responsive element worked correctly and weren't needed.